drivers/filesystems/fastfat_new/fatinit.c

```diff
@@ -15,6 +15,10 @@ Module Name:
 
 #include "fatprocs.h"
 
+#ifdef __REACTOS__
```

Comment:
Why not grouping _Unreferenced_parameter_ and __volatile defines together in a single .h file (for example in the precompiled header) ?

Reply:
I started with a local change, as this define is used there only.
Nonetheless, I could move the latter, if requested.